deploy/staging/ingress.yml

```diff
@@ -9,6 +9,7 @@ metadata:
     nginx.ingress.kubernetes.io/auth-type: basic
     nginx.ingress.kubernetes.io/auth-secret: basic-auth-secret
     nginx.ingress.kubernetes.io/auth-realm: 'Staging User | Authentication Required'
+    nginx.ingress.kubernetes.io/proxy-body-size: "200m"
```

Comment:
Just thinking. Will our PHP-FPM allow this size of upload?

Reply:
It works locally. 
Famous last words!